proxy/http/HttpTransact.h

```diff
@@ -698,16 +698,11 @@ class HttpTransact
     bool dns_round_robin;
     TransferEncoding_t transfer_encoding;
 
-    IpEndpoint addr;    // replaces 'ip' field
-
-    // port to connect to, except for client
-    // connection where it is port on proxy
-    // that client connected to.
-    // This field is managed separately from the port
-    // part of 'addr' above as in various cases the two
-    // are set/manipulated independently and things are
-    // clearer this way.
-    uint16_t port; // host order.
+    // TS-2157 Replace addr with src_addr dst_addr
+    // also replaces port field
+    IpEndpoint src_addr;
+    IpEndpoint dst_addr;
+
```

Comment:
note that uint16_t port is dropped. port is instead stored inside the IpEndpoint struct.